lowrisc-toolchain-gcc-rv32imcb.config

```diff
@@ -25,9 +25,6 @@ CT_CC_GCC_STATIC_LIBSTDCXX=y
 # CT_CC_GCC_LDBL_128 is not set
 CT_CC_LANG_CXX=y
 CT_DEBUG_GDB=y
-CT_GDB_V_8_2=y
-# CT_GDB_CROSS_PYTHON is not set
-CT_ISL_V_0_20=y
```

Comment:
Isn't this version-change related?

Reply:
I wasn't sure and thought I'd give it a try :smile: Turns out CI fails now, so let's bring it back.